techniques/failures/F82.html

```diff
@@ -9,16 +9,23 @@ <h3></h3>
             <p>In the United States, phone numbers are broken into a three digit area code, a three digit prefix, and a four digit extension. A web page creates fixed length text input fields for the three parts of the phone number, surrounding the first field with parenthesis and separating the second and third fields with a dash. Because of this formatting, some users recognize the fields as a phone number. However, there is no text label for the phone number on the web page. This is because the label for each field will be the closest preceding text, so the three fields would be labeled "(", ")" , and "-" respectively.</p>
          
       </section>
-   </section><section id="tests"><h2>Tests</h2>
+   </section>
+    <section id="related">
+         <h2>Related Techniques</h2>
+         <ul>
+          <li><a href="../html/H71">H71</a></li>
+        </ul>
+      </section>
+  <section id="tests"><h2>Tests</h2>
       <section class="procedure"><h3>Procedure</h3>
          <ol>
             <li>For each set of phone number fields in the web page that represents a single phone number, check that the set of fields are labeled with a visible text label that is positioned near the set of phone number fields.</li>
-            <li>For each set of phone number fields in the Web page that represent a single phone number, instructions are provided about how to fill in the fields.</li>
+            <li>For each set of phone number fields in the web page that represent a single phone number, instructions are provided about how to fill in the fields.</li>
```

Comment:
reminder that this change was done simply for consistency with the prior list item's usage of 'web', and line 9's instance of 'web page', as well.

if a larger change to turn this into 'webpage' then there are even more instances that would need to be changed